### PR TITLE
Added New Query  IAM_Password_Without_Valid_Minimum_Length

### LINKED ISSUE #1290: Add  IAM_Password_Without_Valid_Minimum_Length query for Ansible
### Platform
Ansible

### Provider
AWS

### Description
Check if IAM account password policy enforces a minimum length and if that minimum is sufficient.

### PR DESCRIPTION
**Pull request by cataraujo190**
Check if IAM account password policy enforces a minimum length and if that minimum is sufficient. Closes #1290

### CODE REVIEW

File: assets/queries/ansible/aws/iam_password_mininum_length/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "IAM_Password_Without_Valid_Minimum_Length",
+  "queryName": "IAM Password Without Valid Minimum Length",
+  "severity": "MEDIUM",
+  "category": "IAM",
```
Comment (Ruben-Silva) [inline assets/queries/ansible/aws/iam_password_mininum_length/metadata.json:5]: Change the category to "Network Security", and do the same in the corresponding Terraform query, please.

File: assets/queries/ansible/aws/iam_password_mininum_length/query.rego [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  to_number(policyBody.min_pw_length) < 8
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("name={{%s}}.{{community.aws.iam_password_policy}}.{{min_pw_length}}", [policyName]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": "'min_pw_length' is set and no less than 8",
+                "keyActualValue": 	"'min_pw_length' is less than 8"
+              }
+}
```
Comment (Ruben-Silva) [inline assets/queries/ansible/aws/iam_password_mininum_length/query.rego:39]: The attribute 'min_pw_length' has an alias 'minimum_password_length'. Change these queries or add new ones to use when the alias is present.

File: assets/queries/ansible/aws/iam_password_mininum_length/test/positive.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    state: present
+    min_pw_length: 3
+    require_symbols: false
+    require_numbers: true
+    require_uppercase: true
+    require_lowercase: true
+    allow_pw_change: true
+    pw_max_age: 60
+    pw_reuse_prevent: 5
+    pw_expire: false
```
Comment (Ruben-Silva) [inline assets/queries/ansible/aws/iam_password_mininum_length/test/positive.yaml:24]: Add samples with the alias "minimum_password_length", just to be more complete.

**Review by Ruben-Silva — CHANGES_REQUESTED** [review]

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — COMMENTED** [review]
change directory name

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
directory name is wrong

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM (for real now)

**Pull request merged**